lib/features/create-append-anything/CreatePaletteProvider.js

```diff
@@ -0,0 +1,61 @@
+import {
+  assign
+} from 'min-dash';
+import { createIcon } from '../../icons/Icons';
+
+
+/**
+ * A palette provider for the create elements menu.
+ */
+export default function CreatePaletteProvider(palette, translate, popupMenu, canvas) {
+
+  this._translate = translate;
+  this._popupMenu = popupMenu;
+  this._canvas = canvas;
+
+  palette.registerProvider(800,this);
+}
+
+CreatePaletteProvider.$inject = [
+  'palette',
+  'translate',
+  'popupMenu',
+  'canvas'
+];
+
+
+CreatePaletteProvider.prototype.getPaletteEntries = function(element) {
+  var actions = {},
+      translate = this._translate,
+      popupMenu = this._popupMenu,
+      canvas = this._canvas;
+
+  assign(actions, {
+    'create': {
+      group: 'create',
+      imageUrl: createIcon,
+      title: translate('Create element'),
+      action: {
+        click: function(event) {
+          const cursor = event && { x: event.x, y: event.y };
+
+          const position = event && {
+            x: cursor.x + 35,
+            y: cursor.y + 10,
+            cursor
+          };
```

Comment:
Not sure if we want this behavior:

https://user-images.githubusercontent.com/28307541/214012932-6922d29e-96f7-4311-ae74-f07185e3ebc2.mov

Still it's ok to leave it as is if we can't get the entry position easily.